drivers/dummy/rasterizer_dummy.h

```diff
@@ -322,6 +322,8 @@ class RasterizerStorageDummy : public RasterizerStorage {
 		return m->blend_shape_mode;
 	}
 
+	uint32_t mesh_surface_get_stride_in_array(RID p_mesh, int p_surface, int p_array_index) const { return 0; }
```

Comment:
I think I would probably make this function go in VisualServer and not make it virtual, something like:

uint32_t mesh_surface_get_format_stride(int p_format) const;

so it s more or less generic and does not requeire reimplementing in every platform

Reply:
Also does not have the limitation of stalling the render thread if called.

Reply:
ok